cybersecurity/bash-system-monitor/devashmit-day28/monitor.sh

```
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Bash System Monitoring Tool — Day 28 Cybersecurity | Author: devashmit
+# Cron: */5 * * * * /path/to/monitor.sh
+CPU_WARN=70;CPU_CRIT=90;RAM_WARN=75;RAM_CRIT=90;DISK_WARN=80;DISK_CRIT=95
+LOG="/var/log/sysmonitor.log";TS=$(date '+%Y-%m-%d %H:%M:%S')
+RED='\033[0;31m';YEL='\033[0;33m';GRN='\033[0;32m';BOLD='\033[1m';RST='\033[0m'
+colorize(){local v=$1 w=$2 c=$3;if(( $(echo "$v >= $c"|bc -l) ));then echo -e "${RED}${v}%${RST}";elif(( $(echo "$v >= $w"|bc -l) ));then echo -e "${YEL}${v}%${RST}";else echo -e "${GRN}${v}%${RST}";fi;}
```

Shellcheck (0.11.0)

[error] You need a space after the '{'.

(SC1054)